test/freenet/client/filter/CSSParserTest.java

```diff
@@ -657,6 +664,10 @@
 		propertyTests.put("#xy34 { border-style: solid dotted }", "#xy34 { border-style: solid dotted }");
 		propertyTests.put("h1 { border-bottom: thick solid red }", "h1 { border-bottom: thick solid red }");
 		propertyTests.put("h1[foo] { border: solid red; }", "h1[foo] { border: solid red; }");
+		propertyTests.put("div { box-sizing: content-box; }", "div { box-sizing: content-box; }");
+		propertyTests.put("div { box-sizing: border-box; }", "div { box-sizing: border-box; }");
+		propertyTests.put("div { box-sizing: padding-box; }", "div { box-sizing: padding-box; }");
+		propertyTests.put("div { box-sizing: inhemerit; }", "div { }");
```

Comment:
is this a typo? 

    propertyTests.put("div { box-sizing: inhemerit; }", "div { }");